packages/lesswrong/lib/collectionIndexUtils.ts

```diff
@@ -150,3 +131,38 @@ export function combineIndexWithDefaultViewIndex<T extends DbObject>({viewFields
   }
   return combinedIndex;
 }
+
+
+let deferredIndexes: Array<()=>Promise<void>> = [];
+let deferredIndexesTimer: NodeJS.Timeout|null = null;
+
+/**
+ * If running a normal server, defer index creation until 25s after
+ * startup. This speeds up testing in the common case, where indexes haven't
+ * meaningfully changed (but sending a bunch of no-op ensureIndex commands
+ * to the database is still expensive).
```

Comment:
The speed up comes from the fact that they are spaced out from each other, not from changing it to be delayed 10 sec more right? I think that would be helpful to clarify here

What's the purpose of changing the delay to be 10 sec longer anyway?

Reply:
The goal behind the delay was to make it so that in your development cycle, after you make a change, it doesn't happen during the window of time when you're reloading and trying out that change. I was finding that with a 15s delay, it was triggering a lot while I was trying to test the performance of other things.